service/model/release.go

```diff
@@ -109,11 +109,10 @@ func NewReleaseNotification(p Project, r Release) ReleaseNotification {
 }
 
 type GithubRelease struct {
-	ID          int64 // GitHub release ID
-	Name        string
-	Body        string
-	TagName     string  // Git tag associated with the release
-	HTMLURL     url.URL // URL to the release page on GitHub
-	CreatedAt   time.Time
-	PublishedAt time.Time
+	GitTagName string
+	// URL to the release page on GitHub.
+	HTMLURL   url.URL
+	CreatedAt time.Time
+	// Time when the up-to-date information was last fetched from GitHub.
+	UpdatedAt time.Time
```

Comment:
Same here; do we really need these?
(See comment to db migration for detailed reasoning.)

Reply:
As answered above, let's drop it for now.